nodestream/pipeline/extractors/streams/extractor.py

```diff
@@ -83,5 +85,7 @@ async def extract_records(self):
             else:
                 for record in results:
                     yield self.record_format.parse(record)
+        except Exception as e:
+            self.logger.error(f"failed extracting records, Exception: '{e}'")
```

Comment:
We should probably use `self.logger.exception` here. If we just format in the error, then the we will miss some context most likely.

Reply:
ok